chia/timelord/timelord.py

```diff
@@ -335,6 +335,8 @@ async def _reset_chains(self, *, first_run: bool = False, only_eos: bool = False
                     if self.last_state.get_deficit() < self.constants.MIN_BLOCKS_PER_CHALLENGE_BLOCK:
                         self.iters_to_submit[Chain.INFUSED_CHALLENGE_CHAIN].append(new_block_iters)
                     self.iteration_to_proof_type[new_block_iters] = IterationType.INFUSION_POINT
+                    if block in self.overflow_blocks:
+                        self.overflow_blocks.remove(block)
```

Comment:
is it OK remove blocks from `self.overflow_blocks` while iterating over it? Or does `+` create a new copy that we iterate over on line 326?

Reply:
changed it just in case

Reply:
changed